bundle-workflow/src/test_workflow/integ_test/local_test_cluster.py

```diff
@@ -101,6 +102,14 @@ def wait_for_service(self):
         raise ClusterCreationException("Cluster is not green after 10 attempts")
 
     def terminate_process(self):
+        parent = psutil.Process(self.process.pid)
+        logging.info("Checking for child processes")
+        child_processes = parent.children(recursive=True)
+        for child in child_processes:
+            logging.info(f"Found child process with pid {child.pid}")
+            if child.pid != self.process.pid:
+                logging.info(f"Sending SIGKILL to {child.pid} ")
```

Comment:
Same, debug. These don't add much value in the output.

Reply:
Changed to debug